app/views/movies/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1><%= @movie.title %></h1>
+<h3><%= @movie.creation_year %></h3>
+<h3><%= @movie.genre %></h3>
+
+<% @movie.actors_order_of_age.each do |actor| %>
+  <p><%= actor.name %></p>
+<% end %>
+
+<p>Average age of Actors: <%=@movie.average_actor_age.round %></p>
```

Comment:
Consider adding the `.round` to the model method to encapsulate all the logic around finding averages in one place